go/ql/lib/semmle/go/frameworks/Logrus.qll

```diff
@@ -32,7 +32,12 @@ module Logrus {
   }
 
   private class LogCall extends LoggerCall::Range, DataFlow::CallNode {
-    LogCall() { this = any(LogFunction f).getACall() }
+    LogCall() {
+      // find calls to logrus logging functions
+      this = any(LogFunction f).getACall() and
+      // unless all formatters that get assigned are sanitizing formatters
```

Comment:
This isn't quite accurate, since we only check "may-flow" rather than "must-flow".
```suggestion
      // unless all formatters that get assigned may be sanitizing formatters
```